<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EnvGenWidget</class>
 <widget class="QWidget" name="EnvGenWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>552</width>
    <height>481</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Env Partition Operation</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="bottomMargin">
       <number>6</number>
      </property>
      <item>
       <widget class="QGroupBox" name="groupBox_env_sel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="title">
         <string/>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>9</number>
         </property>
         <property name="topMargin">
          <number>9</number>
         </property>
         <property name="rightMargin">
          <number>9</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QRadioButton" name="radioButton_env_gen">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Gen env partition</string>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioButton_env_readback">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Env Readback</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_env_option">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="title">
         <string/>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Platform</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboBox_env_plat">
             <property name="currentText">
              <string>mtk</string>
             </property>
             <item>
              <property name="text">
               <string>mtk</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>Chip</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboBox_env_chip">
             <property name="currentText">
              <string/>
             </property>
             <property name="currentIndex">
              <number>-1</number>
             </property>
             <item>
              <property name="text">
               <string>mt8788</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>mt6833</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>mt6853</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>mt6873</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>mt6877</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>mt6895</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>mt6983</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>mt6895</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Version</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboBox_env_ver">
             <property name="currentIndex">
              <number>-1</number>
             </property>
             <item>
              <property name="text">
               <string>v1</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>v2</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>v3</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>v4</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>v5</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>v6</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>ID</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_env_id">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>0001</string>
             </property>
             <property name="maxLength">
              <number>4</number>
             </property>
             <property name="clearButtonEnabled">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QCheckBox" name="checkBox_env_auto">
             <property name="text">
              <string>Automation</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="checkBox_env_recv">
             <property name="text">
              <string>Recovery</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
